index.html

```diff
@@ -1,417 +1,246 @@
+            <li class="list-item-text"><a href="#portfolio-section" class="link link-text">Portfolio</a></li>
+            <li class="list-item-text"><a href="#About" class="link link-text">About</a></li>
+            <li class="list-item-text"><a href="#contact-section" class="link link-text">Contact</a></li>
+            <li class="list-item-text"><a href="#" class="link link-text fas fa-envelope"></a></li>
+         </ul>
+        <div class="hamburger-menu" id="hamburger-menu"><i class="fas fa-bars"></i>
+      </div>
+    </nav>
+</header>
+    <section class="page-section first-section">
+    <h1 class="section-title title-l1">
+        Hey There. <br>I'm Ben
+    </h1>
+    <h3 class="section-subtitle subtitle-l1">
+        I'am a Software Developer
+    </h3>
+    <div class="intro-details">
+    <p class="description-text">
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don't hestiate to contact me.      
+    </p>   
+    <ul class="social-media-icons ul-list horizontal-list">
+        <li class="list-item-icon"><a href="https://www.github.com/telanet" class="link link-icon fa fa-github"></a></li>
+        <li class="list-item-icon"><a href="https://www.linkedin.com/in/benard-ngetich" class="link link-icon fa fa-linkedin"></a></li>
+        <li class="list-item-icon"><a href="#" class="link link-icon fa fa-angellist"></a></li>
+        <li class="list-item-icon"><a href="https://www.twitter.com/Ben54647605" class="link link-icon fa fa-twitter"></a></li>
+        <li class="list-item-icon"><a href="#" class="link link-icon fab fa-medium-m"></a></li>
+    </ul>
+    </div>
+    </section>
+    <section class="page-section works-section" id="portfolio-section">
+        <div class="box-join-title">
+            <h2 class="title section-title title-1">My Recent Works</h2>
+        <hr class="h-rule">
+        </div>
+          <div class="works-container">
+            <div class="posts-display">
+              <figure class="post-image">
+                <img src="images/img-placeholder-display.png" alt="Placeholder image">
+              </figure>
+              <div class="post-information">
+                <p class="post-title">Multi-Post Stories</p>
+                <p class="post-description description-text"> A daily selection of privately personalized reads; no accounts
+                  or sign-ups required. has been the industry's standard</p>
+                <ul class="post-tags ul-list horizontal-list">
+                  <li class="list-item-text">Html</li>
+                  <li class="list-item-text">Css</li>
+                  <li class="list-item-text">Bootstrap</li>
+                  <li class="list-item-text">Ruby</li>
+                </ul>
+                <button type="button" name="button" class="enable-button post-button">See project</button>
+              </div>
```

Comment:
- [ ] Apology for the mistake of the previous reviewer, kindly stick to the Figma design. In this case, the cards.
## Your Design
![image](https://user-images.githubusercontent.com/86200356/193413662-581bd0c1-39dd-4e9d-bf04-364cb9caae78.png)

## Figma 
![image](https://user-images.githubusercontent.com/86200356/193414068-9f1ba3ba-f8a3-494e-8c88-410e7bc6dea7.png)
You will notice the first card is not there and kindly add the hover and other effects to the button once you add the card.